webrtc.html

```diff
@@ -625,6 +625,313 @@
           </li>
         </ol>
 
+        <p>To <dfn id=
+        "update-ice-gathering-state">update the ice gathering state</dfn> of an
+        <code><a>RTCPeerConnection</a></code> instance <var>connection</var> to
+        <var>newState</var>, the User Agent MUST queue a task that sets
+        <var>connection</var>'s <a href=
+        "#dom-peerconnection-ice-gathering-state">iceGatheringState</a>
+        attribute to <var>newState</var> and fires a simple event named
+        <code><a href=
+        "#event-icegatheringstatechange">icegatheringstatechange</a></code> at
+        <var>connection</var>.</p>
+
+        <p>To <dfn id=
+        "update-ice-connection-state">update the ice connection state</dfn> of an
```

Comment:
I interpreted the Seattle discussions as a consensus to add connectionState, but leave iceConnectionState in for now - we already have people with code that depends on it, and the information it shows isn't *quite* what we want for connectionState.